corehq/motech/dhis2/tasks.py

```diff
@@ -14,7 +14,7 @@
 
 
 @task(queue='background_queue')
-def send_datasets(domain_name, send_now=False):
+def send_datasets(domain_name, send_now=False, send_date=datetime.today()):
```

Comment:
The default value `datetime.today()` will be evaluated once when the function is defined, not every time it's called, and this could have [unexpected consequences][1].  Better to change this to `send_date=None` and then inside the function add
```python
if send_date is None:
    send_date = datetime.today()
```

  [1]: https://pythonconquerstheuniverse.wordpress.com/2012/02/15/mutable-default-arguments/